components/Donation.client.vue

```diff
@@ -19,7 +19,7 @@
     >
       <PrismicText
         wrapper="span"
-        :field="donationData.data['donation-button']"
+        :field="donationData.data['donation_button']"
```

Comment:
I've noticed on our website that the donation button has no label as it was wrongly called here